infrastructure/http/src/main/java/tech/pegasys/teku/infrastructure/http/RestApiConstants.java

```diff
@@ -113,4 +113,19 @@
   public static final String CACHE_NONE = "max-age=0";
   // Finalized max-age equates to 1 year
   public static final String CACHE_FINALIZED = "max-age=31556952";
+
+  public enum CacheLength {
```

Comment:
nit: not really sure this adds much value over string constants above

Reply:
I added it just to move towards more type safety rather than having everything defined as a `String`. Probably should be in the restapi module though so will move it there.